.expeditor/scripts/release_habitat/shared.sh

```diff
@@ -51,7 +60,7 @@ install_release_channel_hab_binary() {
     local pkg_target="${1:-$BUILD_PKG_TARGET}"
     curlbash_hab "${pkg_target}"
 
-    # workaround for https://github.com/habitat-sh/habitat/issues/6771	
+    # workaround for https://github.com/habitat-sh/habitat/issues/6771
```

Comment:
Safe to remove these two lines now?

Reply:
oh, maybe so!

Reply:
Addressed in https://github.com/habitat-sh/habitat/pull/7043/commits/5c2ad84ec14f0c9b01cd6de43de78cbeb52bfe49